app/models/lookup_keys/lookup_key.rb

```diff
@@ -124,6 +124,7 @@ def reject_invalid_lookup_values(attributes)
   end
 
   def default_value_before_type_cast
+    return read_attribute(:default_value) if errors.present? && errors[:default_value].present?
```

Comment:
I think `errors.present?` is redundant here, all ActiveRecord objects will respond true to `.errors.present?` even if there's no error.